src/test/java/com/rocketden/main/service/RoomServiceTests.java

```diff
@@ -347,17 +357,4 @@ public void sendSocketUpdate() {
                 eq(String.format(BaseRestController.BASE_SOCKET_URL + "/%s/subscribe-user", roomDto.getRoomId())),
                 eq(roomDto));
     }
-
-    @Test
-    public void generateValidRoomId() {
-        // Verify room ids are generated correctly
-        String roomId = service.generateRoomId();
-
-        assertEquals(RoomService.ROOM_ID_LENGTH, roomId.length());
-
-        for (char c : roomId.toCharArray()) {
-            assertTrue(c >= '0');
-            assertTrue(c <= '9');
-        }
-    }
```

Comment:
Is this test added elsewhere?

Reply:
It isn't, and it should be -- depending on how we implement the `generateId` method, this could be re-added as it is above, or added in a separate test class.

Reply:
Ok, I implemented this in the `UtilityTests.java` class. I renamed the other file that has the utility methods used for tests to `UtilityTestMethods.java` to better indicate its use.